include/WRAPPER_SLURM

```diff
@@ -1,28 +1,47 @@
 #!/bin/bash
+#SBATCH --job-name="lcdb-wf"
+#SBATCH --partition="niddk,norm"
+#SBATCH --time=24:00:00
 
-if [ -z $SNAKEFILE ]; then
-    SNAKEFILE=$1
-fi
+SNAKEFILE=$1
 
-if [ -z $EXTRA ]; then
+if [ -z $2 ]; then
     EXTRA=$2
+else
+    EXTRA=''
 fi
 
 
+# make logdir
+if [[ ! -e logs ]]; then mkdir -p logs; fi
+
+# Run snakemake
 (source activate lcdb-wf; \
     time snakemake \
     -p \
     -s $SNAKEFILE \
     --directory $PWD \
     -T \
+    -k \
     --rerun-incomplete \
     --jobname "s.{rulename}.{jobid}.sh" \
     -j 999 \
-    --cluster-config clusterconfig.yaml \
+    --cluster-config config/clusterconfig.yaml \
     --verbose \
-    --cluster 'sbatch {cluster.prefix} --cpus-per-task={threads} ' \
+    --cluster 'sbatch {cluster.prefix} --cpus-per-task={threads}  --output=logs/{rule}.o.%j --error=logs/{rule}.e.%j' \
     --use-conda \
-    --configfile config.yml \
+    --configfile config/config.yml \
```

Comment:
No preference. I guess snakemake wrappers expect `environment.yaml`, so let's arbitrarily go with that.